packages/flutter/lib/src/painting/matrix_utils.dart

```diff
@@ -265,9 +265,13 @@ class MatrixUtils {
 List<String> debugDescribeTransform(Matrix4 transform) {
   if (transform == null)
     return const <String>['null'];
-  final List<String> matrix = transform.toString().split('\n').toList();
-  matrix.removeLast();
-  return matrix;
+  final Float64List storage = transform.storage;
+  return <String>[
+    '[0] ${debugFormatDouble(storage[transform.index(0, 0)])},${debugFormatDouble(storage[transform.index(0, 1)])},${debugFormatDouble(storage[transform.index(0, 2)])},${debugFormatDouble(storage[transform.index(0, 3)])}',
```

Comment:
Can you just use `transform.entry(row, column)` instead of getting the index and then accessing storage? Might be a little simpler..